common/js/src/executable-module/emscripten-module.js

```diff
@@ -179,13 +177,13 @@ EmscriptenModule.prototype.getEmscriptenApiModuleSettings_ = function() {
       this.onModuleStderrReceived_(message);
     },
   };
-};
+}
 
 /**
  * Handles the Emscripten module's crash.
  * @private
  */
-EmscriptenModule.prototype.onModuleCrashed_ = function() {
+onModuleCrashed_() {
   goog.log.error(this.logger_, 'The Emscripten module crashed');
   this.dispose();
 };
```

Comment:
nit: I think this should be `}`

Reply:
Done here and below. Thanks for spotting.